### PR TITLE
Bionic UI: power display in kJ, J, mJ

### PR DESCRIPTION
**Pull request by RDru**
#### Summary
```SUMMARY: Interface "Bionic UI: power displayed in kJ, J or mJ"```

#### Purpose of change
Power was always displayed in kJ

#### Describe the solution
Made function that return a string with value and unit.
Named it just "display" for future override for other units.

#### Describe alternatives you've considered
I was thinking about creating units.ccp

#### Testing
Loaded a save, it displays correctly.

#### Additional context
Also changed the description of artificial night generator that I changed in #34456 and forgot about description
obraz

### CODE REVIEW

**Comment by anothersimulacrum** [comment]
#34706 has something very similar

**Comment by RDru** [comment]
> 
> 
> #34706 has something very similar

Mine is about bionic UI window. His is about displaying total power in character panel I think (?).

**Comment by Fris0uman** [comment]
Yep it's the panel.
Can you give a screen shot?

File: src/units.h [hunk 1]
```diff
@@ -488,6 +488,20 @@ inline std::ostream &operator<<( std::ostream &o, const quantity<value_type, tag
     return o << v.value() << tag_type{};
 }
 
+inline const std::string display( const units::energy v )
+{
+    const int kj = units::to_kilojoule( v );
+    const int j = units::to_joule( v );
+    if( kj >= 1 && float( j ) / kj == 1000 ) { // at least 1 kJ and there is no fraction
+        return string_format( "%d kJ", kj );
```
Comment (Night-Pryanik) [inline src/units.h]: I'd like this to be translated.

**Comment by RDru** [comment]
> 
> 
> Can you give a screen shot?

Added screenshot.

File: src/units.h (hunk 1 above)
Comment (RDru) [reply src/units.h]: added translatable units

**Pull request merged**